src/renderer/components/_molecules/data-types/array/index.tsx

```
@@ -87,20 +86,22 @@ const ArrayDataType = ({ data, ...rest }: ArrayDatatypeProps) => {
-                        {type.toLocaleUpperCase()}
-                      </span>
-                    </SelectItem>
-                  )
-                })}
+                <ScrollArea className='max-h-[300px] overflow-y-auto'>
+                  {allTypes.map((type) => {
+                    return (
+                      <SelectItem
+                        value={type}
+                        className='flex w-full cursor-pointer items-center justify-center py-1 outline-none hover:bg-neutral-100 dark:hover:bg-neutral-800'
+                      >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)